R/utils.R

```diff
@@ -317,12 +317,39 @@ un_list <- function(x) {
   names(x) <- nams
   x
 }
-
+# `fmt_txt()` ------------------------------------------------------------------
 #' format strings independent of the cell style.
+#'
+#'
 #' @details
 #' The result is an xml string. It is possible to paste multiple `fmt_txt()`
 #' strings together to create a string with differing styles.
-#' @param x a string or part of a string
+#'
+#' Using `fmt_txt(charset = 161)` will give the Greek Character Set
+#'
+#'  | charset| "Character Set"      |
```

Comment:
Since the charset parameter is documented to be an integer, I removed quotes from the table